ai/integrations/llama_index/unitycatalog-llama_index-with-mlflow.ipynb

```diff
@@ -0,0 +1,996 @@
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Uploading artifacts:   0%|          | 0/14 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Registered model 'ben_wilson.uc_func.llama_index_agent' already exists. Creating a new version of this model...\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "66e1203dae98431bbf0f4ca9e7f0d5d1",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Uploading artifacts:   0%|          | 0/14 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Created version '3' of model 'ben_wilson.uc_func.llama_index_agent'.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "🏃 View run agreeable-moth-983 at: https://e2-dogfood.staging.cloud.databricks.com/ml/experiments/1557189199111224/runs/c99078cdef894beda60dfb74e7576137\n",
+      "🧪 View experiment at: https://e2-dogfood.staging.cloud.databricks.com/ml/experiments/1557189199111224\n"
+     ]
+    }
+   ],
+   "source": [
+    "from mlflow.models import infer_signature\n",
+    "\n",
+    "input_example = {\n",
+    "    \"message\": \"Where is the largest population of Grey Wolves in the contiguous United States? Please answer in 6 words or less.\"\n",
+    "}\n",
+    "\n",
+    "# LLamaIndex's Agent interface is a Pydantic model that is not directly supported by MLflow's signature inference.\n",
+    "# To accomodate the output type received from calling the agent, we need to specify what the response will be\n",
+    "# from the invocation (in this case, a string).\n",
+    "signature = infer_signature(input_example, \"Northern Rocky Mountains, states like Montana.\")\n",
+    "\n",
+    "# The pip requirements defined in the model logging call are needed solely due to using a pre-release build of\n",
+    "# `unitycatalog-ai` and the `unitycatalog-llama_index` packages. When using versions available on PyPI, dependency\n",
+    "# inference should work automatically.\n",
+    "with mlflow.start_run():\n",
+    "    model_info = mlflow.llama_index.log_model(\n",
+    "        \"agent.py\",\n",
+    "        artifact_path=\"model\",\n",
+    "        input_example=input_example,\n",
+    "        signature=signature,\n",
+    "        pip_requirements=[\n",
+    "            \"mlflow\",\n",
+    "            \"git+https://github.com/mlflow/mlflow.git\",\n",
+    "            \"git+https://github.com/unitycatalog/unitycatalog.git#subdirectory=ai/core\",\n",
+    "            \"git+https://github.com/unitycatalog/unitycatalog.git#subdirectory=ai/integrations/llama_index\",\n",
```

Comment:
By the time users find this example we should already released! Maybe just replace to normal package name? Same for the %pip install at the top